### PR TITLE
Retag candidate image after the tests pass

### PR DESCRIPTION
**Pull request by jhadvig**
@mfojtik PTAL

### CODE REVIEW

File: hack/build.sh [hunk 1]
```diff
@@ -67,6 +69,11 @@ for dir in ${dirs}; do
 
   if [[ -v TEST_MODE ]]; then
     IMAGE_NAME=${IMAGE_NAME} test/run
+
+    if [[ $? -eq 0 ]] && [[ "${TAG_ON_SUCCESS}" == "true" ]]; then
+      echo "-> Re-tagging ${IMAGE_NAME} image to ${IMAGE_NAME%"-candidate"}"
+      docker tag $IMAGE_NAME ${IMAGE_NAME%"-candidate"}
```
Comment (bparees) [inline hack/build.sh]: backwards?

**Comment by jhadvig** [comment]
LGTM

File: hack/build.sh (hunk 1 above)
Comment (jhadvig) [inline hack/build.sh]: So the
- `$IMAGE_NAME` - openshift/nodejs-010-centos7-candidate
- `${IMAGE_NAME%"-candidate"}` - openshift/nodejs-010-centos7

bash black magic :sparkles:

Comment (bparees) [inline hack/build.sh]: yup, got it. thanks :)

**Pull request merged**